src/gui/rss/rsswidget.cpp

```diff
@@ -54,6 +54,8 @@
 #include "feedlistwidget.h"
 #include "ui_rsswidget.h"
 
+void convertRelativeUrlToAbsolute(QString &html, const QString &baseUrl);
```

Comment:
It should be inside anonymous namespace as I said before, i.e.:
```c++
namespace
{
    void convertRelativeUrlToAbsolute(QString &html, const QString &baseUrl)
    {
        // code
    }
}
```

Reply:
My misunderstanding of anonymous namespace. I thought it meant an static scope in the source file. I'll correct it later.